xrootd.spec

```diff
@@ -50,8 +50,9 @@ make %makeprocesses VERBOSE=1
 %install
 cd ../build
 make install
+ln -s lib64/cmake/XRootD/XRootDConfig.cmake share/xrootd/cmake/XRootDConfig.cmake
```

Comment:
why this symlink?

Reply:
To keep the file in the old location as well. I can remove that.